roles/cfg_openwrt/templates/corerouter/nftables.d/20-wg-maxseg-size.nft.j2

```diff
@@ -1,7 +1,10 @@
 {% if (networks | selectattr('tunnel_wan_ip', 'defined') | count > 0) and (openwrt_version == 'snapshot') %}
 chain wg_maxseg {
     type filter hook forward priority -1; policy accept;
-	oifname "wg_*" tcp flags syn tcp option maxseg size set meta nfproto map { ipv4 : 1372, ipv6 : 1352 }
-	iifname "wg_*" tcp flags syn tcp option maxseg size set meta nfproto map { ipv4 : 1372, ipv6 : 1352 }
+	{% set min_mtu = ([1412] + (networks | selectattr('tunnel_mtu', 'defined') | map(attribute='tunnel_mtu') | list)) | min %}
```

Comment:
why dont you do the cast to int here?

Furthermore please move the variables in that case to the second line. that will increase readability.